test-environment.yml

```diff
@@ -0,0 +1,31 @@
+# This file describes a conda environment that can be used by alpha testers
+#
+# Run the following command to set up this environment:
+# $ conda env create -f environment.yml
+#
+# The environment name can be overridden with the following command:
+# $ conda env create -n <custom name> -f environment.yml
+#
+# Run the following command to activate the environment:
+# $ source activate cubeviz-testing
+#
+# To deactivate the environment run the following command:
+# $ source deactivate
+#
+# To remove the environment entirely, run the following command:
+# $ conda env remove -n cubeviz-testing
+
+name: cubeviz-testing
+dependencies:
+  - python=3.6
+  - pyqt
+  - numpy
+  - scipy
+  - astropy
+  # This is necessary in some OSX environments to ensure that Python is
+  # installed as a framework
+  - matplotlib
+  - pip:
```

Comment:
Glue has a bunch of other dependencies which will be installed by pip if not included here. I would recommend looking at https://github.com/glue-viz/glue/blob/master/setup.py#L109 and including as many of those as possible.

Reply:
Do you mean including them in the conda dependency section?